keyboards/geminate60/info.json

```diff
@@ -0,0 +1,76 @@
+{
+    "keyboard_name": "geminate60",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
        "LAYOUT_ortho_5x15": {
```

Reply:
This needs to match the layout name in your `h` file.

Reply:
However, this will cause Run QMK lint to fail